content/doc/developer/index.html.haml

```diff
@@ -0,0 +1,39 @@
+---
+title: "Jenkins Developer Guide"
+layout: developer
+---
+
+The Jenkins Developer Guide serves as a reference for core and plugin developers.
+
+%ol
+  - site.devbook.chapters.each do |chapter|
+    %li{:style => 'font-size: 24px;'}
```

Comment:
This kind of thing is better accomplished with `%h3` type tags rather than inline CSS.